src/main/twirl/views/manage/editDeadlinesForm.scala.html

```diff
@@ -0,0 +1,43 @@
+@(field: forms.DateTimeRangeField,
+        opp: Opportunity,
+        questions: Map[String, Question],
+        answers: play.api.libs.json.JsObject,
+        errs: Seq[forms.validation.FieldError],
+        hints: Seq[forms.validation.FieldHint])
+
+
+@errors = @{
+    errs.filter(p => p.path == field.name)
+}
+
+@errors.map { e =>
+    <span class="error-message" role="alert">@e.err</span>
+}
+
+@formatId(id: Long) = @{
+    f"RIFS$id%05d"
+}
+
+@errs
+
+@main("RIFS - Edit Deadline", displayUserName = Some("Portfolio Peter")) {
+
+    <div class="grid-row">
+        <div class="column-two-thirds">
```

Comment:
We probably don't need this grid pattern here - could just use a `<div>` and use class=text on the `<div>` or `<p>`.  I can sort this if you don't mind.